packages/roosterjs-editor-plugins/lib/Paste/fragmentHandler.ts

```diff
@@ -22,10 +23,18 @@ export default function fragmentHandler(doc: HTMLDocument, source: string) {
             // Handle HTML copied from MS Word
             doc.body.innerHTML = html;
             convertPastedContentFromWord(doc);
-        } else if (isWordOnlineWithList(firstNode)) {
+        } else if (firstNode && firstNode.querySelector(WAC_IDENTIFING_SELECTOR)) {
+            // Once it is known that the document is from WAC
+            // We need to remove the display property and margin from all the list item
+            doc.querySelectorAll(WAC_IDENTIFING_SELECTOR).forEach((el: HTMLElement) => {
+                el.style.display = null;
+                el.style.margin = null;
+            });

```

Comment:
I think you can just query once. Here you called both querySelector and querySelectorAll.